### PR TITLE
Support for phpspec 4.0

### PR DESCRIPTION
**Pull request by pamil**
Hello again, another major phpspec release is just around the corner ;)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 language: php
 
-php: [5.6, 7.1, hhvm]
```
Comment (xabbuh) [inline .travis.yml]: What is the reason for this change?

Comment (pamil) [inline .travis.yml]: Phpspec 4 is not tested against HHVM and does not support PHP 5.6, but anyway I should have set the min requirements to PHP 7.0, not PHP 7.1.

Comment (xabbuh) [inline .travis.yml]: Oh, I didn't that you completely forbid phpspec releases before 4.0 now. Isn't it possible to support a constraint like `^3.0|^4.0`?

**Comment by nidup** [comment]
Hi @pamil !

Thank you very much for this pull request :heart_eyes: 

We can follow the same approach we had for phpspec 3.x release:
 - I created a 2.0 branch from the master to maintain this extension 2.0.x supporting phpspec 3
 - The master is now the 3.0-dev branch to prepare the future 3.0.0 release supporting phpsec 4 (cf https://github.com/akeneo/PhpSpecSkipExampleExtension/pull/17)

You can rebase your PR on master, there is still a 7.0 issue, then we'll be able to merge and release a 3.0.0-alpha to test it with projects (i have a good use case for it, as Akeneo PIM master is now using php 7.1 :wink: )

Cheers.

**Comment by pamil** [comment]
Rebased and fixed compatibility with PHP 7.0 🎉

**Comment by nidup** [comment]
Hi @pamil !
Sorry, i missed the notification, thanks a bunch for this PR :tada: 
I merge and prepare the release :dancer:
